app/routes/docs_.schema.$version.$/components.tsx

```diff
@@ -36,9 +36,17 @@ export type Schema = {
   required?: string[]
 }
 
+function formatLinkString(path: string) {
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
```suggestion
function useLinkString(path: string) {
```

Reply:
I don't think that this is resolved. Did you forget to push something?

Reply:
Oops, this time yes, pushed now